spotbugsTestCases/src/java/findhidingsubclasstest/BadInEqualMultipleStaticMethod.java

```diff
@@ -0,0 +1,31 @@
+package findhidingsubclasstest;
+
+class SuperBadInEqualMultipleMethod {
```

Comment:
I don't understand what this class name means.

Reply:
It is the SUPERclass for a non-compliant (BAD) test case, with MULTIPLE METHODs declared in both super and sub classes. But the number methods is INEQUAL. So Super.